src/hooks/use-render.ts

```diff
@@ -53,6 +54,7 @@ const useRender = () => {
     [styleService],
   );
   const isFontLoaded = useWaitForFonts(fonts);
+  const { changeSortOrder, changeActivelySortedHeader } = useSorting(model, layout.qHyperCube);
```

Comment:
Why is this created here? I seems like it can be created in `PivotTable` component instead to avoid a lot of prop drilling?

Reply:
I did like this because we are dealing with **supernova work like `useLoadDataPages`** calls to engine. 
I took the same approach as sn-table where putting a barrier between the work that needs to be done in supernova scope and react scope.

Reply:
We actually do call Engine out side of the "supernova scope". Ex: collapsing or expanding a node and fetching more data when the user scrolls.

So I don't really agree with that as a reason.

Reply:
moved it to Pivot table component